service/tutorial.rest.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- General Guideline Verbs: GET=find, POST=create, PUT=store (create or update), PATCH=update, DELETE=delete  -->
+
```

Comment:
Can we rename the file to order.rest.xml for better naming?

Reply:
Sure, ma'am.

Reply:
Recommended changes updated.
Thank you.